src/pages/sykmeldt/[oppfolgingsdialogId]/arbeidsoppgaver.tsx

```diff
@@ -7,29 +7,40 @@ import {
   Page,
 } from "components/blocks/wrappers/OppfolgingsplanPageSM";
 import { LagredeArbeidsoppgaver } from "components/arbeidsoppgaver/LagredeArbeidsoppgaver";
-import { NyArbeidsoppgave } from "components/arbeidsoppgaver/NyArbeidsoppgave";
+import { NyArbeidsoppgaveSM } from "components/arbeidsoppgaver/NyArbeidsoppgaveSM";
 import { beskyttetSideUtenProps } from "../../../auth/beskyttetSide";
+import { SpacedDiv } from "../../../components/blocks/wrappers/SpacedDiv";
+import Feilmelding from "../../../components/blocks/error/Feilmelding";
 
 const Arbeidsoppgaver: NextPage = () => {
   const aktivPlan = useAktivPlanSM();
 
   const arbeidstakerFnr = aktivPlan?.arbeidstaker.fnr;
-  if (!arbeidstakerFnr) return null;
 
   return (
-    <OppfolgingsplanPageSM page={Page.ARBEIDSOPPGAVER}>
-      {aktivPlan && (
-        <div>
-          <NyArbeidsoppgave />
+    <>
+      {(!aktivPlan || !arbeidstakerFnr) && (
+          <SpacedDiv>
+            <Feilmelding
+              description={
+                "Denne oppfølgingsplanen er ikke aktiv eller mangler arbeidstakers fødselsnummer. Vennligst prøv igjen senere."
```

Comment:
aktivPlan har vi sjekk for inne i OppfolgingsplanPageSM, så den kan fjernes.

Vi har kanskje ikke sjekk for arbeidstakerFnr, så om du vil kan vi legge den til der. Men vet ikke om vi trenger det - det er obligatorisk felt fra backend